GIT_AND_GITHUB.md

```diff
@@ -89,6 +77,13 @@ discuss other alternatives. Of course, this means you are putting yourself at ri
 of sharing your knowledge and/or learning some new stuff, so please be careful not
 to end up being even more awesome than you are.
```

Comment:
That's a pretty good idea. We don't have reviewer's guidelines. Also remember to not take or make this personal, we are all trying to improve each other and build it each other up, not put each other down. 👍